docs/resources/Guild.md

```diff
@@ -854,13 +855,14 @@ Create a new [role](#DOCS_TOPICS_PERMISSIONS/role-object) for the guild. Require
 
 ###### JSON Params
 
-| Field       | Type    | Description                                                    | Default                        |
-| ----------- | ------- | -------------------------------------------------------------- | ------------------------------ |
-| name        | string  | name of the role                                               | "new role"                     |
-| permissions | string  | bitwise value of the enabled/disabled permissions              | @everyone permissions in guild |
-| color       | integer | RGB color value                                                | 0                              |
-| hoist       | boolean | whether the role should be displayed separately in the sidebar | false                          |
-| mentionable | boolean | whether the role should be mentionable                         | false                          |
+| Field       | Type    | Description                                                                                          | Default                        |
+| ----------- | ------- | ---------------------------------------------------------------------------------------------------- | ------------------------------ |
+| name        | string  | name of the role                                                                                     | "new role"                     |
+| permissions | string  | bitwise value of the enabled/disabled permissions                                                    | @everyone permissions in guild |
+| color       | integer | RGB color value                                                                                      | 0                              |
+| hoist       | boolean | whether the role should be displayed separately in the sidebar                                       | false                          |
+| icon        | ?string | the role's [icon hash](#DOCS_REFERENCE/image-formatting) (if the guild has the `ROLE_ICONS` feature) | null                           |
+| mentionable | boolean | whether the role should be mentionable                                                               | false                          |
```

Comment:
```suggestion
| Field       | Type                                     | Description                                                             | Default                        |
| ----------- | ---------------------------------------- | ----------------------------------------------------------------------- | ------------------------------ |
| name        | string                                   | name of the role                                                        | "new role"                     |
| permissions | string                                   | bitwise value of the enabled/disabled permissions                       | @everyone permissions in guild |
| color       | integer                                  | RGB color value                                                         | 0                              |
| hoist       | boolean                                  | whether the role should be displayed separately in the sidebar          | false                          |
| icon        | [image data](#DOCS_REFERENCE/image-data) | the 128x128 role icon image (if the guild has the `ROLE_ICONS` feature) | null                           |
| mentionable | boolean                                  | whether the role should be mentionable                                  | false                          |
```

Reply:
That collidates with https://github.com/discord/discord-api-docs/pull/3815#pullrequestreview-752606768

Reply:
Actually, the recommended size is 64x but there is no size limit afaik, only in terms of file size which is 256kb. Although this can also be a guild emoji so that should also be mentioned I guess